microsoft-edge/webview2/samples/webview2apissample.md

```diff
@@ -229,98 +226,126 @@ The latest version of the WebView2 SDK is now installed for this project.
 
    ![The WebView2APISample app window](./webview2apissample-images/webview2apisample-app-window.png)
 
-1. Use the sample app.
+1. Use the **WebView2APISample** app.
 
 1. In Visual Studio, select **Debug** > **Stop Debugging**.  Visual Studio closes the app.
 
+This completes the numbered steps for building and running the Win32 sample app.  Next, in the Visual Studio code editor, inspect the code, per the following sections.
+
 
 <!-- ====================================================================== -->
-## Step 11 - Inspect the code
+## Hybrid app architecture
+
+The **WebView2APISample** app is an example of a hybrid application, with a Win32 native part and a WebView part.
+*  The Win32 part can directly access native Windows APIs.
+*  The WebView is a container for standard web technologies (HTML, CSS, and JavaScript).
+
+![Hybrid app](./webview2apissample-images/sample-app-layout-diagram.png)
+
+* Section 1: The top part of the **WebView2APISample** app is a Win32 component written in C++. This part of the application takes in UI inputs from the user and uses them to control the WebView.
+
+* Section 2: The main part of the **WebView2APISample** app is a WebView that can be repurposed using standard web technologies (HTML/CSS/JavaScript).  It can be navigated to websites or local content.
+
+This hybrid approach allows you to create and iterate faster using web technologies, while still being able to take advantage of native functionality.  The **WebView2APISample** app demonstrates how the Win32 component and the WebView component can interact with each other.
 
-1. In the Visual Studio code editor, inspect the code, per the following sections.
+This wide-ranging sample app has grown to include 170 menuitems, demonstrating many WebView2 APIs in the Win32/C++ framework.  The present section focuses on basics of hybrid app implementation.
```

Comment:
`...include 170 menuitems...` we should say something like `over 150 menu items` because it may keep increasing as we add more APIs/features.

Reply:
done locally.
This wide-ranging sample app has grown to include over 150 menuitems, demonstrating many WebView2 APIs in the Win32/C++ framework.  The following sections focus on basics of hybrid app implementation.